src/components/BoardUsers/BoardUser.scss

```
@@ -19,21 +19,20 @@
 }
 
 .rest-users {
-  color: $color-backlog-blue;
-  border: 2px solid $color-backlog-blue;
-  height: $icon--extralarge;
-  width: $icon--extralarge;
+  background-color: #333948;
```

```suggestion
  background-color: $menu-icon-background-color--dark;
```

-> `$menu-icon-background-color--dark: #333948;`
Renaming the variable should be considered.